crates/swc_common/src/lib.rs

```diff
@@ -62,7 +62,7 @@ pub mod input;
 pub mod iter;
 pub mod macros;
 pub mod pass;
-#[cfg(feature = "plugin-base")]
+#[cfg(any(feature = "plugin-rt", feature = "plugin-mode"))]
```

Comment:
Is this required?
`plugin-base` is depended by `plugin-rt` and `plugin-mode`.

Reply:
Yes, but looks like i.e if `swc_plugin` sets feature to `plugin-mode` plguin-base cfg won't kick off for those target? also this is more explicitly readable this is supported in both modes.

Reply:
https://github.com/swc-project/swc/blob/d51904b93e4f252e0036bdd1b6ab8ed44836bce4/crates/swc_common/Cargo.toml#L24-L25

`plugin-rt` and `plugin-mode` both enables `plugin-base`.